<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerMlcPositionModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerMlcPositionModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Display</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="ctkCollapsibleButton" name="FooBarCollapsibleButtonWidget">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="qSlicerMlcPositionFooBarWidget" name="FooBar" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="1">
         <widget class="qMRMLSegmentSelectorWidget" name="TargetVolumeSegmentSelectorWidget">
          <property name="segmentationNodeSelectorVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="qMRMLNodeComboBox" name="RTBeamNodeComboBox"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="RTBeamNodeLabel">
          <property name="text">
           <string>Beam node:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="TargetVolumeLabel">
          <property name="text">
           <string>Target volume:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="CalculateMlcPositionPushButton">
        <property name="text">
         <string>Calculate MLC position</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="ShowMlcModelPushButton">
        <property name="text">
         <string>Show MLC model</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentSelectorWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSegmentSelectorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerMlcPositionFooBarWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerMlcPositionFooBarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerMlcPositionModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>RTBeamNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerMlcPositionModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>TargetVolumeSegmentSelectorWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
